bot.py

```diff
@@ -202,8 +202,32 @@ def prefix(self):
             print(Fore.RED + Style.BRIGHT + 'WARNING - The GUILD_ID provided does not exist!' + Style.RESET_ALL)
         else:
             await self.threads.populate_cache()
+        await self.config.update()
 
+        closures = self.config.closures.copy()
 
+        for recipient_id, items in closures.items():
+            after = (datetime.datetime.fromisoformat(items['time']) -
+                     datetime.datetime.utcnow()).total_seconds()
+            if after < 0:
+                after = 0
+            recipient = self.get_user(int(recipient_id))
+
+            thread = await self.threads.find(
+                recipient=recipient)
+
+            if not thread:
+                # If the recipient is gone or channel is deleted
+                self.config.closures.pop(str(recipient_id))
+                await self.config.update()
+                continue
+
+            # TODO: Retrieve messages/replies when bot is down, from history?
```

Comment:
Downtime is only a few seconds anyways I doubt this will be a problem, low priority.